frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx

```diff
@@ -5,32 +5,40 @@ import useSequencedContentCloseHandler from "metabase/hooks/use-sequenced-conten
 import type { HoverCardProps } from "metabase/ui";
 import { HoverCard, useDelayGroup } from "metabase/ui";
 
-export const POPOVER_DELAY: [number, number] = [250, 150];
-export const POPOVER_TRANSITION_DURATION = 150;
+const POPOVER_TRANSITION_DURATION = 150;
+
+// Initially, the user will have to hover for this long to open the popover
+const POPOVER_SLOW_OPEN_DELAY = 1000;
```

Comment:
As a user this is way too long for me.
I'm not used to popovers being displayed only after 1s of hovering.
If I don't see one after ~500ms I'm just going to assume it's not there.

Can we change it to 500, pretty please ? :pray: